python/pyarrow/parquet/core.py

```diff
@@ -3046,7 +3074,8 @@ def read_table(source, *, columns=None, use_threads=True, metadata=None,
             memory_map=memory_map,
             buffer_size=buffer_size,
             coerce_int96_timestamp_unit=coerce_int96_timestamp_unit,
-            decryption_properties=decryption_properties
+            decryption_properties=decryption_properties,
+            page_checksum_verification=page_checksum_verification,
```

Comment:
Same here, this change is not needed.
```suggestion
            decryption_properties=decryption_properties
```

Reply:
Added this too